plugins/image-prioritizer/detect.js

```diff
@@ -51,10 +40,10 @@ function log( ...message ) {
  * @type {InitializeCallback}
  * @param {InitializeArgs} args Args.
  */
-export async function initialize( { isDebug, onLCP } ) {
+export async function initialize( { logger, onLCP } ) {
```

Comment:
See note above passing the logger functions directly without needing the `logger` object.